frontends/concrete-python/tests/execution/test_where.py

```diff
@@ -0,0 +1,132 @@
+"""
+Tests of execution of where operation.
+"""
+
+import random
+
+import numpy as np
+import pytest
+
+from concrete import fhe
+from concrete.fhe.dtypes import Integer
+from concrete.fhe.values import EncryptedScalar, EncryptedTensor
+
+# pylint: disable=redefined-outer-name
+
+functions = [
+    lambda condition, when_true, when_false: np.where(condition, when_true, when_false),
+    lambda condition, when_true, when_false: np.where(condition, when_true, when_false) + 100,
+]
+condition_descriptions = [
+    EncryptedTensor(Integer(is_signed=False, bit_width=1), shape=shape)
+    for shape in [(), (2,), (3, 2)]
+]
+when_true_descriptions = [
+    EncryptedTensor(Integer(is_signed, bit_width), shape=shape)
+    for is_signed in [False, True]
+    for bit_width in [3, 4, 5]
+    for shape in [(), (2,), (3, 2)]
+]
+when_false_descriptions = [
+    EncryptedTensor(Integer(is_signed, bit_width), shape=shape)
+    for is_signed in [False, True]
+    for bit_width in [3, 4, 5]
+    for shape in [(), (2,), (3, 2)]
+]
+chunk_sizes = [
+    2,
+    3,
+]
+
+cases = []
+for function in functions:
+    for condition_description in condition_descriptions:
+        for when_true_description in when_true_descriptions:
+            for when_false_description in when_false_descriptions:
+                for chunk_size in chunk_sizes:
+                    cases.append(
+                        (
+                            function,
+                            condition_description,
+                            when_true_description,
+                            when_false_description,
+                            chunk_size,
+                        )
+                    )
+
+cases = random.sample(cases, 100)
```

Comment:
let's not repeat the same mistakes again with flaky coverage breaks.
can a part of cases be manually chosen to have the coverage right ?

Reply:
It's okay for this case as the code is pretty much branchless except for signed.